Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -2464,13 +2464,14 @@ Generic call to ServiceNow api
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| method | action to be performed on path. Possible values are: GET, POST, PATCH, DELETE. Default is 0. | Required | 
-| path | the API path starting with forward slash (/). | Required | 
-| json_body | whether or not the request body is json. Possible values are: true, false. Default is false. | Optional |
-| body | json to send in body. | Optional | 
-| headers | json of headers to add. | Optional | 
+| method | Action to be performed on path. Possible values are: GET, POST, PATCH, DELETE. Default is 0. | Required | 
+| path | The API path (without a leading forward slash '/'). | Required | 
```

Comment:
These changes are only to make the descriptions a bit more uniform. In my last PR I learned, that these descriptions should start with a capital letter and words like json or id should be capitalized (therefore json -> JSON and capitalized the first character of each description here). 
Because I needed to add the new custom api argument I uniformed the other arguments at the same time.

The path argument was corrected because it wrongly stated that the argument must start with a leading slash (see discussion below)

Reply:
Please revert this change.